src/analytics/private/load-ods-data-workflow.ts

```diff
@@ -177,15 +176,142 @@ export class LoadOdsDataToRedshiftWorkflow extends Construct {
     });
 
     //
-    // function:  createLoadRedshiftTableWorkflow
+    // Refresh materialized views
+    //
+    const refreshViewsFn = this.refreshViewsFn(props, copyRole);
+    const refreshViewsStep = new LambdaInvoke(this, `${this.node.id} - Refresh materialized views`, {
+      lambdaFunction: refreshViewsFn,
+      resultSelector: {
+        'Payload.$': '$.Payload',
+      },
+      resultPath: '$.refreshViewsOut',
+    });
+    refreshViewsStep.addRetry({
+      errors: ['Lambda.TooManyRequestsException'],
+      interval: Duration.seconds(3),
+      backoffRate: 2,
+      maxAttempts: 6,
+    });
+
+    //
+    // Next state machines
+    //
+    let nexTaskExecChain;
+    for (const nexTask of props.nextStateStateMachines) {
+      const taskName = nexTask.name;
+      const stateMachine = nexTask.stateMachine;
+      const input = nexTask.input;
+      const integrationPattern = nexTask.integrationPattern?? IntegrationPattern.REQUEST_RESPONSE;
+      const resultPath = `$.${nexTask.resultPath ?? taskName.replace(/ /g, '')}`;
+      const nexTaskExec = new StepFunctionsStartExecution(this, `${this.node.id} - ${taskName}`, {
```

Comment:
specify below parameters to specify the execution name of sub workflow,

```json
{
  "StateMachineArn": "state arn",
  "Input": {
    "AWS_STEP_FUNCTIONS_STARTED_BY_EXECUTION_ID.$": "$$.Execution.Id"
  },
  "Name.$": "States.Format('{}-{}', States.ArrayGetItem(States.StringSplit($$.Execution.Id, ':'), 7), 'my-suffix')"
}
```

Reply:
Fixed